src/fetch/transfer_file.py

```diff
@@ -118,7 +92,7 @@ def auto_propose(dune: DuneFetcher, slack_client: WebClient, dry_run: bool) -> N
 if __name__ == "__main__":
     args = generic_script_init(description="Fetch Complete Reimbursement")
     args.dune.log_saver.print(
-        f"The data being aggregated here is available for visualization at\n"
+        f"The data aggregated can be visualized at\n"
```

Comment:
Changing the tense of this phrase since by the time anyone sees this message the data has already been aggregated.